.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,28 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior.
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**System information (please complete the following information):**
+- Operating system: (`python -c 'import platform; print(platform.platform())'`)
+- Python version: (`python -V`)
+- pyMOR version: (`python -c 'import pymor; print(pymor.__version__)'`)
```

Comment:
```suggestion
**System information:**
Output of `python -c 'import pymor; print(pymor.config)'`.
```

With #1483, both the Python version and the OS version are included in pymor.config.